code/cellular_automaton/src/elementary_cellular_automata/ElementaryCellularAutomaton.java

```diff
@@ -65,11 +65,15 @@ private int getCellIndex(int cell) {
 	
 	@Override
 	public String toString() {
-		StringBuilder builder = new StringBuilder();
+		String result = "";
```

Comment:
Do you need the ` = ""`?

Reply:
Yes, because otherwise the compiler complains.